src/rest/main/main.go

```diff
@@ -95,17 +104,17 @@ func main() {
 		}
 	}()
 
-	if caFile == "" && server.ClientAuth.Enabled("cert") {
+	if caFile == "" && clientAuth.Enabled("cert") {
 		glog.Fatal("Must specify -cacert with -client_auth cert")
 	}
 
 	swagger.Load()
 
 	server.GenerateJwtSecretKey()
-	server.JwtRefreshInt = time.Duration(jwtRefInt * uint64(time.Second))
-	server.JwtValidInt = time.Duration(jwtValInt * uint64(time.Second))
+	server.JwtRefreshInt = time.Duration(30 * uint64(time.Second))
```

Comment:
FWIW, you don't need to cast to uint64 here, the Go compiler should have automatically handled it.

Reply:
fixed